electronics_lib/JlcResistor.py

```diff
@@ -66,24 +68,24 @@ def __init__(self, **kwargs):
 
   def select_resistor(self, resistance: Range, power_dissipation: Range,
                       part_spec: str, footprint_spec: str) -> None:
-    part = JLcResistorTable.table().filter(lambda row: (
-            (not part_spec or part_spec == row[JLcResistorTable.PART_NUMBER]) and
-            (not footprint_spec or footprint_spec == row[JLcResistorTable.FOOTPRINT]) and
-            row[JLcResistorTable.RESISTANCE].fuzzy_in(resistance) and
-            power_dissipation.fuzzy_in(row[JLcResistorTable.POWER_RATING])
+    part = JlcResistorTable.table().filter(lambda row: (
+            (not part_spec or part_spec == row[JlcResistorTable.PART_NUMBER]) and
+            (not footprint_spec or footprint_spec == row[JlcResistorTable.FOOTPRINT]) and
+            row[JlcResistorTable.RESISTANCE].fuzzy_in(resistance) and
+            power_dissipation.fuzzy_in(row[JlcResistorTable.POWER_RATING])
     )).first(f"no resistors in {resistance} Ohm, {power_dissipation} W")
 
-    self.assign(self.selected_resistance, part[JLcResistorTable.RESISTANCE])
-    self.assign(self.selected_power, part[JLcResistorTable.POWER_RATING])
-    self.assign(self.lcsc_part, part[JLcTable.DISTRIBUTER_PART_NUMBER])
+    self.assign(self.selected_resistance, part[JlcResistorTable.RESISTANCE])
+    self.assign(self.selected_power, part[JlcResistorTable.POWER_RATING])
+    self.assign(self.lcsc_part, part[JlcTable.DISTRIBUTER_PART_NUMBER])
 
     self.footprint(
-      'R', part[JLcResistorTable.FOOTPRINT],
+      'R', part[JlcResistorTable.FOOTPRINT],
       {
         '1': self.a,
         '2': self.b,
       },
-      mfr=part[JLcResistorTable.MANUFACTURER], part=part[JLcResistorTable.PART_NUMBER],
-      value=f"{part[JLcTable.DESCRIPTION]}",
-      datasheet=part[JLcResistorTable.DATASHEETS]
+      mfr=part[JlcResistorTable.MANUFACTURER], part=part[JlcResistorTable.PART_NUMBER],
+      value=f"{part[JlcTable.DESCRIPTION]}",
```

Comment:
```suggestion
      value=part[JlcTable.DESCRIPTION],
```

Don't need the f-string / string interpolation if it's just one variable